regression/cbmc/gcc_bswap1/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+\[main\.assertion\..\] assertion __builtin_bswap(16|32|64)\((sb|b|bl)\) == 0: FAILURE$
```

Comment:
could you put the assertion number so that it is clear which ones are supposed to fail

Reply:
Done.